### ISSUE TITLE
To sensitive to touch when deactivating via flocking the card

### ISSUE THREAD

**Issue by Klaus385385** [rpetrich/ProSwitcher#280]

**Comment by rpetrich** [comment]
Agreed, on iPad it's twice as sensitive as it should be. Fixed for 1.2alpha2

**Comment by Eatonfl** [comment]
I'm on 1.2alpha3 and I find the sensitivity perfect. I was always a bit frustrated with how slow it was on my iPod touch 3rd gen. I like how on alpha 3 I can just flick without pausing
